prqlc/prqlc-parser/src/lexer.rs

```diff
@@ -35,6 +35,7 @@ pub enum Token {
     Or,          // ||
     Coalesce,    // ??
     DivInt,      // //
+    Pow,         // **
```

Comment:
If we just comment this out, but leave literally everything else, I think that'll be fine. (I do think we should prioritize correctness, so I would vote against merging `**` when it doesn't get the associativity right)